### PR TITLE
Putting into our new format for cloud

### PR DESCRIPTION
**Pull request by moxiegirl**
This is a carry for PR 12323 which basically I accidentally merged when my hand slipped on my track ball mouse.  Rather than revert, I'm gonna just do this.

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

File: docs/sources/installation/amazon.md [hunk 1]
```diff
@@ -2,52 +2,16 @@ page_title: Installation on Amazon EC2
...
-own) and skip the step with the *User Data*. Then continue with the
-[*Ubuntu*](/installation/ubuntulinux) instructions.
-
-Continue with the [User Guide](/userguide/).
+## Amazon EC2
+
+You can install Docker on any AWS EC2 Amazon Machine Image (AMI) that runs an
+operating system which Docker supports. Amazon's website includes specific
+instructions for [installing on Amazon
+Linux](https://docs.docker.com/installation/#installation). To install on a
```
Comment (thaJeztah) [inline docs/sources/installation/amazon.md]: spurious ` a `

File: docs/sources/installation/amazon.md [hunk 2]
```diff
@@ -2,52 +2,16 @@ page_title: Installation on Amazon EC2
...
-on any EC2 instance running Ubuntu. Just follow Step 1 from the Amazon
-QuickStart above to pick an image (or use one of your
-own) and skip the step with the *User Data*. Then continue with the
-[*Ubuntu*](/installation/ubuntulinux) instructions.
-
-Continue with the [User Guide](/userguide/).
+## Amazon EC2
+
+You can install Docker on any AWS EC2 Amazon Machine Image (AMI) that runs an
+operating system which Docker supports. Amazon's website includes specific
```
Comment (thaJeztah) [inline docs/sources/installation/amazon.md]: s/which/that/ ? Or "operating system supported by Docker"

File: docs/sources/installation/amazon.md [hunk 3]
```diff
@@ -2,52 +2,16 @@ page_title: Installation on Amazon EC2
...
+instructions for [installing on Amazon
+Linux](https://docs.docker.com/installation/#installation). To install on a
+another AMI, follow the instructions for its specific operating
+system in this installation guide. 
+
+For detailed information on Amazon AWS support for Docker, refer to [Amazon's
+documentation
+here](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.
```
Comment (thaJeztah) [inline docs/sources/installation/amazon.md]: "here" may be redundant.

**Comment by moxiegirl** [comment]
ping @docker/docs-owners

File: docs/sources/installation/amazon.md [hunk 4]
```diff
@@ -2,52 +2,15 @@ page_title: Installation on Amazon EC2
...
+
+You can install Docker on any AWS EC2 Amazon Machine Image (AMI) which runs an
+operating system that Docker supports. Amazon's website includes specific
+instructions for [installing on Amazon
+Linux](https://docs.docker.com/installation/#installation). To install on
+another AMI, follow the instructions for its specific operating
+system in this installation guide. 
+
+For detailed information on Amazon AWS support for Docker, refer to [Amazon's
+documentation](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.
```
Comment (thaJeztah) [inline docs/sources/installation/amazon.md]: tiny-pico-nit: `http://` -> `https://` for the amazon link (no show-stopper)

**Comment by thaJeztah** [comment]
Thanks! One comment, but nothing to block merging, Looks fine to me :)

File: docs/sources/installation/amazon.md [hunk 5]
```diff
@@ -2,52 +2,15 @@ page_title: Installation on Amazon EC2
...
-own) and skip the step with the *User Data*. Then continue with the
-[*Ubuntu*](/installation/ubuntulinux) instructions.
-
-Continue with the [User Guide](/userguide/).
+## Amazon EC2
+
+You can install Docker on any AWS EC2 Amazon Machine Image (AMI) which runs an
+operating system that Docker supports. Amazon's website includes specific
+instructions for [installing on Amazon
+Linux](https://docs.docker.com/installation/#installation). To install on
```
Comment (fredlf) [inline docs/sources/installation/amazon.md]: The way this is phrased, I expect the link to go to Amazon's website.

Comment (moxiegirl) [inline docs/sources/installation/amazon.md]: Good catch @fredlf it should be.  I'll fix that.

**Comment by SvenDowideit** [comment]
LGTM

**Pull request merged**